### PR TITLE
hehe

### PR DESCRIPTION
**Pull request by shreyans264**

### CODE REVIEW

File: js files/a.js [hunk 1]
```diff
@@ -2,13 +2,13 @@ function myFunction() {
     var x, text;
 
     // Get the value of the input field with id="numb"
-    x = document.getElementById("numb").value;
+    x = document.getElementById("numb").value
```
Comment (shreyans264) [inline js files/a.js]: Missing semicolon

Comment (shreyans264) [inline js files/a.js]: Missing semicolon

File: js files/a.js [hunk 2]
```diff
@@ -2,13 +2,13 @@ function myFunction() {
...
 
     // If x is Not a Number or less than one or greater than 10
     if (isNaN(x) || x < 1 || x > 10) {
         text = "Input not valid";
     } else {
-        text = "Input OK";
```
Comment (shreyans264) [inline js files/a.js:11]: Missing semicolon

File: js files/a.js [hunk 3]
```diff
@@ -2,7 +2,7 @@ function myFunction() {
     var x, text;
 
     // Get the value of the input field with id="numb"
-    x = document.getElementById("numb").value;
+    x = document.getElementById("numb").value
```
Comment (robo-cop) [inline js files/a.js]: Missing semicolon

File: js files/a.js [hunk 4]
```diff
@@ -2,13 +2,13 @@ function myFunction() {
     var x, text;
 
     // Get the value of the input field with id="numb"
-    x = document.getElementById("numb").value;
```
Comment (robo-cop) [inline js files/a.js]: Nice change

File: js files/a.js [hunk 5]
```diff
@@ -2,11 +2,11 @@ function myFunction() {
...
+    x = document.getElementById("numb").value
```
Comment (robo-cop) [inline js files/a.js]: Missing semicolon

Comment (robo-cop) [inline js files/a.js]: Missing semicolon

Comment (robo-cop) [inline js files/a.js]: Missing semicolon

Comment (robo-cop) [inline js files/a.js]: Missing semicolon

Comment (robo-cop) [inline js files/a.js]: Missing semicolon

File: c.css [hunk 6]
```diff
@@ -0,0 +1,1599 @@
...
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    height:20px;
+}
+
+.patientHeader{
+    padding-top:25px;
+    padding-bottom:10px;
+    padding-left:20px !important;
```
Comment (robo-cop) [inline c.css]: Try to avoid !important

File: c.css [hunk 7]
```diff
@@ -0,0 +1,1599 @@
...
+    color:#6f6f6f;
+}
+#patient_root h2{
+    font-weight: 600;
+    font-size:18px;
+}
+.selectpatient-button{
+    border-right: 1px solid #CCC;
+    padding: 0 7px;
+    color: #577996 !important;
```
Comment (robo-cop) [inline c.css]: Try to avoid !important

File: c.css [hunk 8]
```diff
@@ -0,0 +1,1599 @@
...
+    background-color: #e7f5fc;
+    -webkit-box-shadow: 0 1px 4px rgba(0, 0, 0, 0.065);
+    -moz-box-shadow: 0 1px 4px rgba(0, 0, 0, 0.065);
+    box-shadow: 0 1px 4px rgba(0, 0, 0, 0.065);
+    border:1px solid #a4c5d5;
+    color:#6989a3;
+    margin-left: -7px;
+}
+#patient_header_container .medicalhistory a{
+    height: 20px !important;
```
Comment (robo-cop) [inline c.css]: Try to avoid !important

File: c.css [hunk 9]
```diff
@@ -0,0 +1,1599 @@
...
+    line-height: 20px !important;
```
Comment (robo-cop) [inline c.css]: Try to avoid !important

File: c.css [hunk 10]
```diff
@@ -0,0 +1,1599 @@
...
+    margin: 5px 0 !important;
```
Comment (robo-cop) [inline c.css]: Try to avoid !important

File: c.css [hunk 11]
```diff
@@ -0,0 +1,1599 @@
...
+    background: #FFF;
+}
+#patient_header_container .medicalhistory a i{
+    margin: 2px 0 0 -5px;
+}
+#patient_header_container .medicalhistory a.black{
+    background: #444;
+    color: #FFF !important;
```
Comment (robo-cop) [inline c.css]: Try to avoid !important

File: c.css [hunk 12]
```diff
@@ -0,0 +1,1599 @@
...
+}
+.drugs_row input.input-mini{
+    width:38px;
+}
+#medical .drugs_row input,#medical .drugs_row select{
+    margin-top:5px;
+    margin-bottom:5px;
+}
+.delete_row_link{
+    padding:0px !important;
```
Comment (robo-cop) [inline c.css]: Try to avoid !important

File: c.css [hunk 13]
```diff
@@ -0,0 +1,1599 @@
...
+    margin: 0px !important;
```
Comment (robo-cop) [inline c.css]: Try to avoid !important

File: c.css [hunk 14]
```diff
@@ -0,0 +1,1599 @@
...
+    position: relative !important;
```
Comment (robo-cop) [inline c.css]: Try to avoid !important

File: c.css [hunk 15]
```diff
@@ -0,0 +1,1599 @@
...
+    padding: 15px 14px 10px;
+}
+
+.popover.right .arrow {
+    top: 17px;
+}
+
+
+.cancelled td, .cancelled td *, .timeline_items .cancelled{
+    background: #efefef !important;
```
Comment (robo-cop) [inline c.css]: Try to avoid !important

File: c.css [hunk 16]
```diff
@@ -0,0 +1,1599 @@
...
+    color:#cfcfcf !important;
```
Comment (robo-cop) [inline c.css]: Try to avoid !important

File: c.css [hunk 17]
```diff
@@ -0,0 +1,1599 @@
...
+#patient-documents .popover.right {
+    margin-left: 10px;
+    background-color: #555;
+    opacity: .9;
+    color: #fff;
+    font-size: 1.1em;
+}
+
+.cancelled td.cancelled_reason{
+    color:#000 !important;
```
Comment (robo-cop) [inline c.css]: Try to avoid !important

File: c.css [hunk 18]
```diff
@@ -0,0 +1,1599 @@
...
+.quick .btn-small{
+    margin-top:1px;
+}
+
+.item.child{
+    margin-left: 10px;
+}
+
+#patient_header_container .green{
+    color:#608d15 !important;
```
Comment (robo-cop) [inline c.css]: Try to avoid !important

File: c.css [hunk 19]
```diff
@@ -0,0 +1,1599 @@
...
+
+.print-patient-info{
+    font-size: 1.2em;
+    padding-left:30px;
+    padding-bottom: 10px;
+    border-bottom: 1px solid #ccc;
+}
+.sticky_bottom .chzn-drop{
+    bottom: 26px;
+    top: auto !important;
```
Comment (robo-cop) [inline c.css]: Try to avoid !important

File: c.css [hunk 20]
```diff
@@ -0,0 +1,1599 @@
...
+}
+.info-block select.value{
+    width: 214px;
+}
+.info-block .errors{
+    margin-left: 204px;
+    margin-top: 5px;
+}
+.info-block input.error, select.error{
+    background: #FBE3E4 !important;
```
Comment (robo-cop) [inline c.css]: Try to avoid !important

File: c.css [hunk 21]
```diff
@@ -0,0 +1,1599 @@
...
+    border-color: #C0392B !important;
```
Comment (robo-cop) [inline c.css]: Try to avoid !important

File: c.css [hunk 22]
```diff
@@ -0,0 +1,1599 @@
...
+}
+.area.newceb .discountblock input{
+    width: 25%;
+}
+.area.newceb select{
+    margin-bottom: 0;
+}
+.area.newceb .delete{
+    line-height: 20px;
+    color: #F65B5B !important;
```
Comment (robo-cop) [inline c.css]: Try to avoid !important

File: c.css [hunk 23]
```diff
@@ -0,0 +1,1599 @@
...
+}
+
+.area.newceb .dropdown-toggle.pseudolink{
+    border: none;
+    font-size: 13px;
+    width: 90px;
+    text-align: left;
+    font-family: 'OpenSans', 'Arial', 'Verdana', sans-serif;
+    padding: 0;
+    margin-bottom: -3px !important;
```
Comment (robo-cop) [inline c.css]: Try to avoid !important

File: c.css [hunk 24]
```diff
@@ -0,0 +1,1599 @@
...
+}
+.listview .table tbody td.soap_data_content {
+	border-bottom:1px solid #ccc;
+}
+.listview .table tbody tr:nth-last-child(3) td.soap_data_content:last-child {
+	border-bottom: none;
+}
+
+.rx-minder-icon-top-margin {
+    margin-top: 6px !important;
```
Comment (robo-cop) [inline c.css]: Try to avoid !important

File: c.css (hunk 6 above)
Comment (robo-cop) [inline c.css]: Try to avoid !important

File: c.css (hunk 7 above)
Comment (robo-cop) [inline c.css]: Try to avoid !important

File: c.css (hunk 8 above)
Comment (robo-cop) [inline c.css]: Try to avoid !important

File: c.css (hunk 9 above)
Comment (robo-cop) [inline c.css]: Try to avoid !important

File: c.css (hunk 10 above)
Comment (robo-cop) [inline c.css]: Try to avoid !important

File: c.css (hunk 11 above)
Comment (robo-cop) [inline c.css]: Try to avoid !important

File: c.css (hunk 12 above)
Comment (robo-cop) [inline c.css]: Try to avoid !important

File: c.css (hunk 13 above)
Comment (robo-cop) [inline c.css]: Try to avoid !important

File: c.css (hunk 14 above)
Comment (robo-cop) [inline c.css]: Try to avoid !important

File: c.css (hunk 15 above)
Comment (robo-cop) [inline c.css]: Try to avoid !important

File: c.css (hunk 16 above)
Comment (robo-cop) [inline c.css]: Try to avoid !important

File: c.css (hunk 17 above)
Comment (robo-cop) [inline c.css]: Try to avoid !important

File: c.css (hunk 18 above)
Comment (robo-cop) [inline c.css]: Try to avoid !important

File: c.css (hunk 19 above)
Comment (robo-cop) [inline c.css]: Try to avoid !important

File: c.css (hunk 20 above)
Comment (robo-cop) [inline c.css]: Try to avoid !important

File: c.css (hunk 21 above)
Comment (robo-cop) [inline c.css]: Try to avoid !important

File: c.css (hunk 22 above)
Comment (robo-cop) [inline c.css]: Try to avoid !important

File: c.css (hunk 23 above)
Comment (robo-cop) [inline c.css]: Try to avoid !important

File: c.css (hunk 24 above)
Comment (robo-cop) [inline c.css]: Try to avoid !important

**Pull request closed without merging**